packages/notification-center/src/shared/config/themeDefaultValues.ts

```diff
@@ -60,9 +57,16 @@ export const defaultDarkTheme: INovuTheme = {
   },
   footer: { logoTextColor: colors.white, logoPrefixFontColor: colors.B40 },
   loaderColor: colors.vertical,
-  unseenBadge: { color: { fillColor: { stopColor: '#FF512F', stopColorOffset: '#DD2476' }, borderColor: colors.B15 } },
 };
 
 export const defaultCommonTheme: ICommonTheme = {
-  fontFamily: 'Montserrat',
```

Comment:
@djabarovgeorge Looks like this is the. reason for the failing test. Should this be Lato instead of Monsterrat?

Reply:
Yea i updated it, it is 'Montserrat' due the test from the past i update all, and added a comment below regarding it.